docs/pages/setup/admin/daemon.mdx

```diff
@@ -45,17 +67,83 @@ PIDFile=/run/teleport.pid
 WantedBy=multi-user.target
 ```
 
-## Daemon restarts
+<Notice type="tip">
+
+If `/usr/lib/systemd/system/` does not exist, consult the list of
+[unit file load paths](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Unit%20File%20Load%20Path)
+for other supported paths.
+
+</Notice>
+
+Enable the unit so systemd can place it in its dependency tree:
+
+```code
+$ sudo systemctl enable teleport.service
+Created symlink /etc/systemd/system/multi-user.target.wants/teleport.service → /lib/systemd/system/teleport.service.
+```
+
+Restart your Linux host and log in again. You can confirm that Teleport is
```

Comment:
A full restart seems unnecessary, but I suppose it does demonstrate that the Teleport service will start on machine boot, so I'm a little conflicted.